test/functionalTest/cases/0876_entity_dates/entity_dates_overriden_by_user.test

```diff
@@ -0,0 +1,415 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Entity dates overriden by the user
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# Steps 4 to 7 are weird, but this is the way CB works now (and it is documented in
```

Comment:
the way CB works now ....  Seems pretty definitive.

the way CB works **right** now => seems like it might change in the future

Reply:
Fixed in commits 4274c664e11888879464ab84fde8d3f34768bdaf...ad4c3fcfee9b12d1b3ba0bfa62491b562f3cdabd